File: 2.0/js/calls/getCallRecordingMp3ByName.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.calls.getCallRecordingMp3ByName({
      id: 379506003,
      name: 'call-recording1'
    })
      .then((response) => {
        // returns binary response
        console.log(response.data);
      })
      .catch((err) => {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/calls/getCallRecordingMp3ByName.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.getCallRecordingMp3ByName(
    id=379506003,
    name='call-recording1'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)